_data/finance.yml

```diff
@@ -163,6 +163,7 @@ websites:
       sms: Yes
       email: Yes
       software: Yes
+      doc: https://ttlc.intuit.com/questions/2902682-what-is-two-step-verification
```

Comment:
I think this url `https://ttlc.intuit.com/questions/2905185` explains the process better than that one.

Reply:
Sure, looks good to me.